packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -372,10 +378,14 @@ export default decorate([
         defaultFilter,
         filter: value = defaultFilter,
         log,
-      }) =>
-        value === 'all'
-          ? log.tasks
-          : filter(log.tasks, ({ status }) => status === value),
+        page,
+      }) => {
+        const start = (page - 1) * ITEMS_PER_PAGE
+        const tasks = log.tasks.slice(start, start + ITEMS_PER_PAGE)
```

Comment:
it will not be called in this case, Please see https://github.com/vatesfr/xen-orchestra/blob/56db0df756a90f406b225f7b0b76010be6f1961e/packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js#L447